src/data/snippet/use-key-press-enter.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: 'useKeyPressEnter'
+summary: 'Hooks to help creating key press handler'
+topic: 'react'
+author: 'Abui'
+github_username: 'abui-am'
+created_at: '10/01/2022'
+---
+
+## TypeScript Implementation
+
+```ts
+import { KeyboardEvent } from 'react'
+
+const useKeyPressEnter = (onPressEnter: () => void) => {
+  function keyHandler(event: KeyboardEvent<HTMLDivElement>): void {
+    if (event.key === 'Enter') {
+      if (onPressEnter) onPressEnter()
+    }
+  }
+  return keyHandler
+}
+
+export { useKeyPressEnter }
+```
+
+## Usage
+
+```
```

Comment:
hi @abui-am
you might want to add file extension like `.ts` or `.js` to this code block 👍🏻